azure-pipelines.yml

```diff
@@ -91,6 +92,7 @@ jobs:
 
   - task: DockerCompose@0
     displayName: 'Build: Dev'
+    condition: and(succeeded(), ne(variables['build.reason'], 'PullRequest'))
```

Comment:
Would you mind giving brief overview of what this change provides?

Reply:
I followed the pattern. We were only pushing to dev acr if this condition was met, therefore it seemed to me that we don't need to build it unless that condition is met.
As per the [docs](https://docs.microsoft.com/en-us/azure/devops/pipelines/process/conditions?view=azure-devops&tabs=yaml) the condition means that this step/task will only trigger if the build (previous steps ) succeeded and the reason behind this build is not a PullRequest.